src/main/resources/node1/application.conf

```diff
@@ -5,7 +5,7 @@ ergo {
   # Settings for node view holder regime. See papers.yellow.ModifiersProcessing.md
   node {
     # Keep state root hash only and validate transactions via ADProofs
-    ADState = false
+    stateType = "utxo"
```

Comment:
If the set of possible states known beforehand I would consider adding proper comment here. Otherwise people will find out about it only when app crashes.

Reply:
good idea!